api/desecapi/tests/test_dyndns12update.py

```diff
@@ -296,6 +296,191 @@ def test_subnet(self):
         self.assertEqual(response.data, "good")
         self.assertIP(ipv6="2a01::3303:72dc:f412:7233", subname="foo")
 
+    def test_update_multiple_v4(self):
+        # /nic/update?hostname=a.io,sub.a.io&myip=1.2.3.4
+        new_ip = "1.2.3.4"
+        domain1 = self.my_domain.name
+        domain2 = "sub." + self.my_domain.name
+
+        with self.assertRequests(
```

Comment:
why is there no request regarding `domain2`?

Reply:
Because `domain2` is a subdomain of `domain1`, so it's not a cross-domain update and done atomically in one backend transaction.

The other case (cross-domain) is not supported by our backend and tested in `test_update_multiple_with_subdomains_of_different_domains` (returns `400`).